config/initializers/doorkeeper.rb

```diff
@@ -25,7 +25,20 @@
       if current_level_of_authentication_is_good_enough
         current_user
       else
-        raise LevelOfAuthenticationTooLowError
+        destination_url =
```

Comment:
Do we need a feature flag for this?

Reply:
No, because `current_level_of_authentication_is_good_enough` is true if the feature flag is disabled.

Reply:
ace